docs/guide/env-and-mode.md

```diff
@@ -66,7 +66,7 @@ interface ImportMetaEnv {
 
 ## Modes
 
-By default, the dev server (`serve` command) runs in `development` mode, and the `build` command runs in `production` mode.
+By default, the dev server (`dev` command) runs in `development` mode, and the `build` command runs in `production` mode.
```

Comment:
There is no `dev` nor `serve` command:

```suggestion
By default, the dev server (the `vite` command) runs in `development` mode, and the `vite build` command runs in `production` mode.
```

Reply:
well I'd say the npm commands were intended here and those are the ones by default:
```json
"scripts": {
  "dev": "vite",
  "build": "vite build",
  "serve": "vite preview"
},
```

Reply:
That's from the `create-app` templates. This is the guide to Vite itself, and there's `vite build` referenced below on line 71/85, for example.

Reply:
Makes sense